### ISSUE TITLE
Screen Reader: Heading level 2 should be the immediate focus after creating a new problem set

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#689]
After selecting the palettes, as a screen reader user, I am immediately directed to the editable text area. However, the directions are located in the heading level 2

Focus should be placed at "Heading Level 2: Type or edit the equation using your..." 

((tested using voiceover)
